### ISSUE TITLE
"nothing but" and similar

### ISSUE THREAD

**Issue by nschneid** [UniversalDependencies/UD_English-EWT#272]
Indefinite pronoun + but:

- [2 of these 10 occurrences](http://match.grew.fr/?corpus=UD_English-EWT@dev&custom=61b625dd4b1a5&clustering=rel.label) treat "but" as a conjunction and pronoun as the first conjunct
  * GUM: [1 of 4 occurrences](http://match.grew.fr/?corpus=UD_English-GUM@dev&custom=61b627e6b4dc5&clustering=rel.label) using coordination. Note that indefinite pronouns are NOUN in GUM.
- "do anything but" [analyzed as coordination with "do"](http://match.grew.fr/?corpus=UD_English-EWT@dev&custom=61b6261639f49)

I think the `nmod` analysis is better.

**Comment by nschneid** [comment]
Note that in "Your answer doesn't do anything but remind me...", "anything" is an NPI. (Unlike "I wanted to do anything but...".) The "remind me" part doesn't belong in the scope of negation—should it be an `obl`?

**Comment by amir-zeldes** [comment]
Agreed, the conj case is an error, will fix in GUM. For "anything but remind me" I agree it should be `advcl` - essentially it's like "except", and we also tag these "but"s as prepositions (so does OntoNotes).

**Comment by nschneid** [comment]
(Documented [here](https://universaldependencies.org/en/dep/fixed.html))
